### DISCUSSION TITLE
Allow Toxin Cabinets to be linked to (Medical) Fabricator.

### DISCUSSION THREAD

**Discussion by TheDarkBox** [FakeFishGames/Barotrauma#7076] [Ideas and suggestions]
For some reason, Toxin Cabinets can't be linked to Medical Fabricators, which is rather strange, considering Secure Steel Cabinets (which seem to be functionally the same, save for the sprite) can be linked to fabricators of any sort.

**Comment by 3e849f2e5c** [comment]
This was such a simple addition that I've gone ahead and implemented it in https://github.com/Regalis11/Barotrauma-development/commit/35a17ea4cab90adfa41ab54963f315252c4872af
